src/Aspire.Dashboard/Extensions/StringComparers.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Dashboard.Extensions;
+
+internal static class StringComparers
+{
+    public static StringComparer ResourceType => StringComparer.Ordinal;
+}
```

Comment:
This is a convention I've seen used in a few places, giving semantic names to different domains of strings, ensuring they're compared in a uniform way.